gdx/src/com/badlogic/gdx/math/Matrix4.java

```diff
@@ -1201,54 +1252,54 @@ static inline void matrix4_rot(float* mat, float* vec) {
 	 * array where the x-component of the first vector is located. The numVecs parameter specifies the number of vectors stored in
 	 * the vectors array. The stride parameter specifies the number of floats between subsequent vectors and must be >= 3. This is
 	 * the same as {@link Vector3#mul(Matrix4)} applied to multiple vectors.
-	 * @param mat the matrix
-	 * @param vecs the vectors
-	 * @param offset the offset into the vectors array
+	 * @param mat     the matrix
+	 * @param vecs    the vectors
+	 * @param offset  the offset into the vectors array
 	 * @param numVecs the number of vectors
 	 * @param stride the stride between vectors in floats */
-	public static native void mulVec (float[] mat, float[] vecs, int offset, int numVecs, int stride) /*-{ }-*/; /*
+	public static native void mulVec(float[] mat, float[] vecs, int offset, int numVecs, int stride) /*-{ }-*/; /*
```

Comment:
This is not following libGDX's code style.

Reply:
We usually pass the code through Spotless.
Maybe Spotless is not formatting that case i.e. it's nox fixing all spaces on method definitions.
As side note, Spotless is adding an extra newline in comments.